build.sh

```diff
@@ -0,0 +1,2 @@
+docker build -t ghmeier/coinage ./
```

Comment:
nah, the docker hub images are a separate repo from github ones [https://hub.docker.com/r/ghmeier/coinage/](https://hub.docker.com/r/ghmeier/coinage/)